components/service/glean/src/main/java/mozilla/components/service/glean/Glean.kt

```diff
@@ -358,7 +358,7 @@ open class GleanInternalAPI internal constructor () {
      * @param pings List of pings to send.
      * @return The async Job performing the work of assembling the ping
      */
-    internal fun sendPings(pings: List<PingType>) = Dispatchers.API.launch {
+    fun sendPings(pings: List<PingType>) = Dispatchers.API.launch {
```

Comment:
Why is this change needed?

Reply:
Reverted, it was before I figured out how to send a custom ping. 😄